### DISCUSSION TITLE
Unordered list bullets

### DISCUSSION THREAD

**Discussion by wolispace** [wbrowar/craft-3-guide#113] [Ideas and Feature Requests]
Im trying this, and we are getting correct nesting, but the disc is not showing as I assume its not supported yet:

```html
  <ul class="g-list-disc g-list-outside g-ml-6 g-space-y-2">
    <li class="g-pl-2">Sitemap - Single Page</li>
    <li class="g-pl-2">Landing Page - Single Page</li>
    <li class="g-pl-2">News and Event content</li>
    <li class="g-pl-2">Special Event</li>
  <ul>
```

Is there another way of getting the disc shown?

**Comment by wbrowar** [comment]
Hi @wolispace, in Guide 4 I had deprecated the guide utility classes feature (and then removed it in Guide 5), mainly for this type of situation. I was looking for a way to make Tailwind work with PHP to read from your guides and then pull out more utility classes based on what you wanted to use, but the dependencies to make that work were fragile and complicated (at the time I was looking into it).

I wrote a blog post regarding this when Guide 5 came out, so please check that out if you'd like to hear more about the reason behind it:
https://wbrowar.com/article/code/guide-5-bts-slideouts-vue-and-web-components-and-plugin-design#it-was-a-good-idea-at-the-time

For now, since you seem to be using Guide 3 or 4, what I would recommend doing is to add a class and using the `{% css %}` tag to add your own custom CSS to the guide. If you decide to upgrade to Guide 5 (which requires Craft 5), then there's more support for writing CSS in your guide (including CSS selectors for each guide and a new CSS code editor field).

The Control Panel in Craft CMS has a handful of helpful CSS Custom Properties, but I'd really like to see the CP have some sort of built-in solution for working with utility classes in the future.

**Comment by wolispace** [comment]
Ok, we are probably a year+ away from being in a position to upgrade to Craft 5 so the `{% css %}` workaround will do.

Thanks for clarifying.
